tests/ui/debug_assert_with_mut_call.stderr

```diff
@@ -0,0 +1,68 @@
+error: Do not call a mutable function inside of a debug_assert!
+  --> $DIR/debug_assert_with_mut_call.rs:27:5
+   |
+LL |     debug_assert!(foo_bool_mut(&mut 3));
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: `#[deny(clippy::debug_assert_with_mut_call)]` on by default
+note: Invocation here
+  --> $DIR/debug_assert_with_mut_call.rs:27:19
+   |
+LL |     debug_assert!(foo_bool_mut(&mut 3));
+   |                   ^^^^^^^^^^^^^^^^^^^^
+   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

Comment:
I'd like to annotate the whole macro and espacyiall the mut call, but I think this is too verbose. Any better idea?

Reply:
See review comment above: IMO it is sufficient, if only the function call is annotated. No need for a extra `note`, since this is, as you said, too verbose.

Reply:
See review comment below: IMO it is sufficient to just annotate the function/method call.